.buildkite/gen-pipeline.sh

```diff
@@ -178,10 +178,12 @@ run_mpi_integration() {
     run_test "${test}" "${queue}" \
       ":tensorflow: MPI Keras MNIST (${test})" \
       "bash -c \"${oneccl_env} \\\$(cat /mpirun_command) python /horovod/examples/keras/keras_mnist_advanced.py\""
+  fi
 
+  if [[ ${test} != *"torch1_2"* ]]; then
```

Comment:
We are adding a few more test runs here as we did not run it for tf2 before, now we do. This is expected?

Reply:
Yeah, in my mind if the Docker has TF 2 and PyTorch 1.3, we should still run it, based off the version of PyTorch.